packages/block-editor/src/components/block-draggable/draggable-chip.js

```diff
@@ -22,7 +23,7 @@ export default function BlockDraggableChip( { clientIds } ) {
 			);
 
 			if ( ! isOfSameType ) {
-				return layout;
+				return stack;
```

Comment:
Is this being used anywhere? Seems like we can remove this dependency if we don't display an icon for multiple blocks.

Reply:
Addressed this one, good catch thanks.